package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -537,254 +792,296 @@ def _update_water_selection(self):
                 'name {0}'.format(' '.join(self.acceptors)))
             self.logger_debug("Water acceptors: {0}".format(len(self._water_acceptors)))
 
-    def run(self, start=None, stop=None, step=None, verbose=None, debug=None):
-        """Analyze trajectory and produce timeseries.
+    def _get_bonded_hydrogens(self, atom):
```

Comment:
For the future, shouldn't we rather use bond information (if available) and only fall back to a distance search if no bonds are available?

Reply:
@orbeckst Thanks for the information. I think I'm not keeping up with the MDAnalysis. I wodner if it is possible to point out the corresponding information, please? Thank you.

Reply:
I wonder if there is any update on this matter? Or shall I update this part when this PR is merged?

Reply:
Leave for another PR.